docs/src/main/sphinx/connector/kudu.rst

```diff
@@ -15,6 +15,14 @@ The Kudu connector is compatible with all Apache Kudu versions starting from 1.0
 If the connector uses features that are not available on the target server, an error is returned.
 Apache Kudu 1.8.0 is currently used for testing.
 
+Requirements
+------------
+
+To connect to Kudu, you need:
+
+* Kudu version 1.15.0 or higher.
```

Comment:
Not sure about that version .. In the above paragraph it says Kudu 1.8.0 is used for testing. Lets update to that for now.

Reply:
We use 1.10 for testing at the moment. 

https://github.com/trinodb/trino/blob/97af1beea554f324390a6cb6755e351b246494ad/plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestingKuduServer.java#L40